src/SymbolProvider/StructuredDebugDirectorySymbolProviderTest.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/match.h>
+#include <gtest/gtest.h>
+
+#include "OrbitBase/Future.h"
+#include "OrbitBase/ImmediateExecutor.h"
+#include "OrbitBase/StopSource.h"
+#include "SymbolProvider/ModuleIdentifier.h"
+#include "SymbolProvider/StructuredDebugDirectorySymbolProvider.h"
+#include "SymbolProvider/SymbolLoadingOutcome.h"
+#include "Test/Path.h"
+#include "TestUtils/TestUtils.h"
+
+using orbit_test_utils::HasValue;
+
+namespace orbit_symbol_provider {
+
+constexpr SymbolLoadingSuccessResult::SymbolSource kSymbolSource =
+    SymbolLoadingSuccessResult::SymbolSource::kStadiaSymbolStore;
+
+TEST(StructuredDebugDirectorySymbolProvider, RetrieveSymbols) {
```

Comment:
I would suggest to split this into three different test cases: RetrieveSymbolsSuccess, RetrieveSymbolsNotFound and RetrieveSymbolsError.
The style guides suggest that "Test names should summarize the behavior being tested and its expected outcome." 
See go/unit-testing-practices#naming

Reply:
Thats a good idea! done